Cargo.toml

```diff
@@ -1,32 +1,7 @@
-[package]
-name = "centrifuge-chain"
-version = "0.10.35"
-authors = ["Centrifuge <admin@centrifuge.io>"]
-description = "Centrifuge chain implementation in Rust."
-build = "build.rs"
-edition = "2021"
-license = "LGPL-3.0"
-default-run = "centrifuge-chain"
-homepage = "https://centrifuge.io/"
-repository = "https://github.com/centrifuge/centrifuge-chain"
```

Comment:
All of this should stay, otherwise our `workspace.*` is empty right?!

Reply:
Yes, actually, they are under `[workspace.package]` section.